tests/global_config_x86_64.py

```diff
@@ -44,6 +44,7 @@
 instance_type_centos_os_matrix = generate_linux_instance_type_os_matrix(
     os_name="centos.stream", preferences=[CENTOS_STREAM9_PREFERENCE, CENTOS_STREAM10_PREFERENCE]
 )
+instance_type_fedora_os_matrix = generate_linux_instance_type_os_matrix(os_name="fedora", preferences=["fedora"])
```

Comment:
this is also needed under global_config_arm64.py, right?

Reply:
+1, use constants available.

Reply:
but you marked the test as @pytest.mark.arm64
either do no mark or add to global_config_arm64.py

Reply:
You are right.
Removed the mark